filters/auth/oidc.go

```diff
@@ -38,8 +41,33 @@ const (
 	stateValidity       = 1 * time.Minute
 	oidcInfoHeader      = "Skipper-Oidc-Info"
 	cookieMaxSize       = 4093 // common cookie size limit http://browsercookielimits.squawky.net/
+
+	// Deprecated: The host of the Azure Active Directory (AAD) graph API
+	azureADGraphHost = "graph.windows.net"
 )
 
+var distributedClaimsClients = sync.Map{}
```

Comment:
Because of runtime host eval here I think we need to be thread safe

Reply:
The host is not known on filter creation as far as I understand, so seems to be right to use sync.Map or a Mutex, yes